app/vector_icons/BUILD.gn

```diff
@@ -7,30 +7,53 @@ import("//brave/components/speedreader/common/buildflags/buildflags.gni")
 import("//build/config/ui.gni")
 import("//components/vector_icons/vector_icons.gni")
 
+action_foreach("leo_icons") {
+  script = "//brave/app/vector_icons/convert_svg_to_icon.py"
+
+  # These should be the names Leo Icons
+  # They will be available in C++ as |kLeo{IconName}Icon|
+  icons = [
+    "arrow-right.svg",
+    "heart-filled.svg",
+    "heart-outline.svg",
+    "rss.svg",
+  ]
+
+  leo_icons_base_path = "../../node_modules/@brave/leo/icons"
+  sources = rebase_path(icons, ".", leo_icons_base_path)
```

Comment:
totally optionl:

how about using `inputs`? Though sources and inputs are equivalent in actions, but I think inputs are going well with convert_svt_to_icon.py's param name.

Reply:
It seems like `action_foreach` expects `sources` rather than `inputs` - when I change it I get this:
![image](https://user-images.githubusercontent.com/7678024/220190795-2f5e9b55-040d-447b-b183-888f37470d08.png)

Reply:
Oops, sorry for the wrong comment. Thank you for trying!

Reply:
Nah all good! It definitely would've been more semantic if it worked :smile: